packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -59,20 +66,31 @@ module.exports = class ImportObjectsAction extends BaseAction {
 
 		try {
 			const flags = [];
+			let scriptIdArray;
 			if (this._runInInteractiveMode) {
 				if (params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS] !== undefined && !params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[ANSWERS_NAMES.IMPORT_REFERENCED_SUITESCRIPTS];
 				}
+
+				scriptIdArray = params[ANSWERS_NAMES.SCRIPT_ID].split(' ');
 			} else {
 				this._log.info(NodeTranslationService.getMessage(WARNINGS.OVERRIDE));
 				if (params[COMMAND_FLAGS.EXCLUDE_FILES]) {
 					flags.push(COMMAND_FLAGS.EXCLUDE_FILES);
 					delete params[COMMAND_FLAGS.EXCLUDE_FILES];
 				}
+
```

Comment:
@karl-anthony-ng the scriptid in importobjects behaves like "contains" and not "equals"
so if a user provides a scriptid, and there are alot of matching scriptids returned, this can also break the output.